content-elements/tce-modal/edit/index.vue

```diff
@@ -23,10 +23,14 @@
 </template>
 
 <script>
+import cloneDeep from 'lodash/cloneDeep';
 import { EmbeddedContainer } from 'tce-core';
+import EventBus from 'EventBus';
 import Preview from './Preview';
 import values from 'lodash/values';
 
+const appChannel = EventBus.channel('app');
```

Comment:
See prev comment regarding event-bus exposure. We need to find another way of addressing this problem.